exercises/perfect-numbers/example.rb

```diff
@@ -0,0 +1,39 @@
+class PerfectNumber
+  VERSION = 1
+  attr_accessor :num
+  
+  def initialize(num)
+    raise "#{num} is not a natural number." if num < 0
```

Comment:
I see! Sorry for not being clear enough! I was suggesting that we could write a test to see if an exception will be raised in case we initialize this class with something other than a natural number.